### DISCUSSION TITLE
spaCy for Python 2.7?

### DISCUSSION THREAD

**Discussion by GlowTube** [explosion/spaCy#7673] [Help: Installation] [answered]
Project requirements lock me into using Ubuntu 18.04 / Python 2.7.17.

- What's the last version of spaCy that will run on this system?
- My needs will probably be fairly simple (parsing, tagging of English snippets). Will I hit some sort of "wall" forced to stick with the older version (meaning, is it foolhardy to do this for some reason)?

Thanks!

**Comment by svlandeg** [comment, marked as answer]
Hi! 

If you're locked into Python 2.7, you can still use spaCy 2.x, with the latest release `v2.3.5`: https://github.com/explosion/spaCy/releases/tag/v2.3.5

spaCy v3 only supports Python 3. But spaCy v2.3.5 should be stable and good for you to use for the use-cases you've summed up. So I hope that works for you!
